[D1] Fix Order of D1 `changelog.md` by Date (#9702)
cloudflare/cloudflare-docs: content/d1/changelog.md

D1 currently exposes read & writes per second, query response size, and query latency percentiles.

## 2023-06-16

### Generated columns documentation

> We've published new documentation describing how to use D1's support for [generated columns](/d1/learning/generated-columns/) to define columns that are dynamically generated on write (or read). Generated columns allow you to extract data from [JSON objects](/d1/learning/querying-json/) or use the output of other SQL functions.

Reviewer: ```suggestion
New documentation has been published on how to use D1's support for [generated columns](/d1/learning/generated-columns/) to define columns that are dynamically generated on write (or read). Generated columns allow you to extract data from [JSON objects](/d1/learning/querying-json/) or use the output of other SQL functions.
```
